pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -512,7 +512,7 @@ func Convert_v1_PersistentVolumeClaim_To_api_Disk(name string, disk *api.Disk, c
 
 // Convert_v1_Hotplug_PersistentVolumeClaim_To_api_Disk converts a Hotplugged PVC to an api disk
 func Convert_v1_Hotplug_PersistentVolumeClaim_To_api_Disk(name string, disk *api.Disk, c *ConverterContext) error {
-	if c.IsBlockDV[name] {
+	if c.IsBlockPVC[name] {
```

Comment:
can we create a unit test case in `pkg/virt-launcher/virtwrap/converter/converter_test.go` for this?

@awels maybe you'd have input on how to achieve that?

Reply:
There is a hotplug context in the converter_test.go but it is currently concerned with the scsi controller. We could add some tests that populate the IsBlockPVC and isBlockDV maps and call this function.